src/Homework5/decor_result.py

```diff
@@ -0,0 +1,27 @@
+"""
+2. Создайте декоратор, который хранит результаты вызовы функции
+(за все время вызовов, не только текущий запуск программы)
+"""
+
+from datetime import datetime
+from runner_funcs import nod_evklid
+from runner_funcs import runner
+
+
+def decor_result(func):
+    def wr(*args, **kwargs):
+        with open('results.txt', 'a') as f:
+            f.write('Я функция: {} '
```

Comment:
it has troubles with encoding (you can check your file in pr)

Reply:
It was win-1251 encoding in file, no idea, how it happens ;-), switch to utf-8